crates/rpc-types-eth/src/filter.rs

```diff
@@ -14,47 +15,50 @@ use itertools::{
     Itertools,
 };
 
-/// Helper type to represent a bloom filter used for matching logs.
-#[derive(Debug, Default)]
-pub struct BloomFilter(Vec<Bloom>);
+/// FilterSet is a set of values that will be used to filter logs.
+#[derive(Clone, Debug, PartialEq, Eq)]
+#[cfg_attr(feature = "serde", derive(serde::Deserialize))]
+pub struct FilterSet<T: Eq + Hash> {
+    set: HashSet<T>,
```

Comment:
I believe this deserde impl no longer behaves like before, but unsure where this is even used, but we should treat this as serde transparent

Reply:
`serde(from = "HashSet<T>")` should cover this, ya?